aca_preview/preview.py

```diff
@@ -180,6 +187,17 @@ def add_review_methods(cls, aca):
         aca.context = {}  # Jinja2 context for output HTML review
         aca.messages = []  # Warning messages
 
+    @property
+    def is_OR(self):
+        """Return ``True`` if obsid corresponds to an OR."""
+        obsid = float(self.obsid)
+        return abs(obsid) < 38000
```

Comment:
I spoke with Steve and John about this and I think we might be better off with just obsid < 38000 .  The negative target lists are large (on order of 7000 for this cycle).

Reply:
I'm going to merge this and open a new PR for this issue.